linkerd/app/core/src/errors/respond.rs

```diff
@@ -388,7 +388,7 @@ impl<R> Respond<R> {
 
 impl<B, R> respond::Respond<http::Response<B>, Error> for Respond<R>
 where
-    B: Default + hyper::body::HttpBody,
+    B: Default + linkerd_proxy_http::Body,
```

Comment:
:point_up: an example of replacing `hyper::body::HttpBody` with a stable `linkerd_proxy_http::Body`.